content/docs/04-clusters/03-edge/11-deploy-cluster.md

```diff
@@ -0,0 +1,1141 @@
+|Manifests|Add new manifest, and name it `hello-universe`|
+
+When you provide the value `hello-universe` in the **Manifest** field, a blank text editor will open on the righthand side of the screen. Copy the following manifest and paste it into the text editor. 
+<br />
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: hello-universe-service
+spec:
+  type: NodePort
+  selector:
+    app: hello-universe
+  ports:
+  - protocol: TCP
+    port: 8080
+    targetPort: 8080
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hello-universe-deployment
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: hello-universe
+  template:
+    metadata:
+      labels:
+        app: hello-universe
+    spec:
+      containers:
+      - name: hello-universe  
+        image: ghcr.io/spectrocloud/hello-universe:1.0.12 
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 8080
+```
+
+The screenshot below shows the manifest pasted into the text editor. Click on the **Confirm & Create** button to finish adding the manifest. 
+
+  ![A screenshot of Hello Universe application manifest.](/tutorials/edge/clusters_edge_deploy-cluster_add-manifest-file.png)
+
+
+If there are no errors or compatibility issues, Palette will display the newly created full cluster profile for review. Verify the layers you added, and click on the **Next** button. 
+<br />
+
+Review all layers and click **Finish Configuration** to create the cluster profile. 
+<br />
+
+## Create a Cluster
+
+Click on the newly created cluster profile to view its details page.  Click the **Deploy** button to deploy a new Edge cluster. 
+<br />
+
+![Screenshot of the Profile Layers success.](/tutorials/edge/clusters_edge_deploy-cluster_profile-success.png)
+
+The cluster deployment wizard will display the following sections. 
+<br />
+
+### Basic Information
+
+Use the following values in the first section, **Basic Information**. 
+
+|**Field**|**Value**|
+|---|---|
+|Cluster name| docs-tutorial-cluster |
+|Description| Cluster as part of the Edge tutorial.|
+|Tags|`spectro-cloud-education, app:hello-universe, terraform_managed:false`|
+
+Click **Next** to continue.
+<br /> 
+
+### Parameters
+
+The **Parameters** section offers you another opportunity to change the profile configurations. For example, clicking on the **BYOS Edge OS 1.0.0** layer allows you to configure the `system.registry`, `system.repo`, and other available attributes. 
+
+Use the default values for all attributes across all layers and click **Next**.
+<br />
+
+### Cluster config
+
+Provide the Virtual IP (VIP) address for the host cluster to use during the cluster configuration process. You can optionally select an SSH key to remote into the host cluster and provide a Network Time Protocol (NTP) server list, if available, with you.  
+
+Click **Next** to continue.    
+<br />
+
+### Nodes config
+
+In this section, provide details for the master pool. 
```

Comment:
Spend some time here explaining that you are using the Edge hosts to make up the cluster nodes. One acts as the control plane node, and the other two as the worker nodes. This is a critical point of the learning journey.